Header header
uint8_t led_index
uint8_t strip_id
uint8_t led_r
uint8_t led_g
uint8_t led_b